tests/story-variants.spec.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2016-2023 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { expect, test } from '@playwright/test';
+import * as fs from 'fs';
+
+const storyIds: string[] = JSON.parse(fs.readFileSync('./dist/docs/stories.json').toString());
+
+for (const storyId of storyIds) {
+  if (storyId.includes('--variants') === false) {
```

Comment:
The return value of `includes` is a boolean, so a strict check here isn't necessary. 

`storyId.includes('--variants') === false` => `!storyId.includes('--variants')`

Reply:
I am used to doing `=== false` from my previous job. We had a rule not to use the `!` since it's easy to miss. I can change it, though.

Reply:
This is changed now.